crypto/s2n_certificate.c

```diff
@@ -326,7 +326,7 @@ int s2n_cert_chain_and_key_load_pem(struct s2n_cert_chain_and_key *chain_and_key
     POSIX_ENSURE_REF(chain_and_key);
 
     POSIX_GUARD(s2n_cert_chain_and_key_set_cert_chain(chain_and_key, chain_pem));
-    POSIX_GUARD(s2n_cert_chain_and_key_set_private_key(chain_and_key, private_key_pem));
+    //POSIX_GUARD(s2n_cert_chain_and_key_set_private_key(chain_and_key, private_key_pem));
```

Comment:
This should be left as is. Instead I believe we should add `s2n_cert_chain_load_pem` in order to only import certificate data to the `struct s2n_cert_chain_and_key` object.